### PR TITLE
PLANET-7071 Rewrite Take Action Boxout test in Playwright

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-7071](https://jira.greenpeace.org/browse/PLANET-7071)
This test now contains two use cases, one for custom fields and one for existing page fields. This PR also removes the old acceptance test.

### Testing

You can run the test on local either with the VS Code extension if you have it, or in a terminal by running `npx playwright test tests/e2e/take-action-boxout.spec.js`

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [deimos](https://www-dev.greenpeace.org/test-deimos) | [admin](https://www-dev.greenpeace.org/test-deimos/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-deimos/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-deimos) | [composer-local.json](https://github.com/greenpeace/planet4-test-deimos/blob/main/composer-local.json)

:watch: 2023.05.12 09:18:47

File: tests/e2e/tools/lib/login.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { nonce } from './nonce';
 
-async function login( page, context ) {
```
Comment (mleray) [inline tests/e2e/tools/lib/login.js:3]: The `page` parameter wasn't used

File: tests/e2e/tools/lib/login.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { nonce } from './nonce';
```
Comment (GP-Dan-Tovbein) [inline tests/e2e/tools/lib/login.js:1]: Can we remove the empty spaces to follow the `eslint` rules?

File: tests/e2e/tools/lib/new-post.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { login } from './login';
```
Comment (GP-Dan-Tovbein) [inline tests/e2e/tools/lib/new-post.js:1]: Same here!

**Review by GP-Dan-Tovbein — APPROVED** [review]
Looks good to me! I've tested locally and passed both tests 👏 

I've just some comments but doesn't block the approval.

**Pull request merged**